### ISSUE TITLE
Startup errors unless worker indexing is disabled for .NET 8

### ISSUE THREAD

**Issue by mattchenderson** [Azure/azure-functions-dotnet-worker#2185]
I wasn't sure if this warranted a separate issue but my function app is in East US 2 (using `FUNCTIONS_WORKER_RUNTIME = dotnet-isolated` + `netFrameworkVersion = v8.0` + `use32BitWorkerProcess = false`) and exhibiting the exact same error exceptions described by the OP in [#9686](https://github.com/Azure/azure-functions-host/issues/9686)

I managed to get around this by using `<FunctionsEnableWorkerIndexing>False</FunctionsEnableWorkerIndexing>` in my Functions csproj

Mentioning here as 9686 was closed with a request to follow on here.

_Originally posted by @vexingpos in https://github.com/Azure/azure-functions-dotnet-worker/issues/2075#issuecomment-1872333050_

**Comment by mattchenderson** [comment]
Hi @vexingpos - could you please add the following information as well? Thanks!

- What version of the core worker dependencies ([Microsoft.Azure.Functions.Worker](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker/) and 
[Microsoft.Azure.Functions.Worker.Sdk](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Sdk/)) are you using?
- For hosted instances where you're seeing this behavior, what is the host version that you are seeing used?
- Can you confirm the app is not hosted in an App Service Environment (ASE)?
- If this is happening locally, what version of the Core Tools are you using?
- Have you tried this in other regions? If so, was there any difference from East US 2?

And I know you mentioned seeing the same error as in another issue, but can you please share the specific traces or exception messages that you are seeing?

**Comment by vexingpos** [comment]
I haven't been able to reproduce this issue since, but at the time of the issue, I was using these core packages

```
"Microsoft.Azure.Functions.Worker" Version="1.20.0"
"Microsoft.Azure.Functions.Worker.Sdk" Version="1.16.3"
```

Hosted on a regular App Service. Current runtime version is **4.28.4.21900** but this will likely have changed since my first issue report.

Works locally.

**Comment by kshyju** [comment]
Thanks for the update @vexingpos 
It is possible that your app was deployed to a region which did not get updated with a hosting config changes we applied (to enable the worker indexing feature e2e) in December. The issue must have been resolved after that hosting config deployment was done.

Going to mark this as closed as OP confirmed everything is working fine now. But if you ever run into issues with dotnet isolated function apps, don't hesitate to open a new issue with relevant details. Thank you!
